### ISSUE TITLE
[Bug]: The default Enderman texture and animations are still being overridden despite the option has already disabled 

### ISSUE THREAD

**Issue by asrieldashie** [bonsaistudi0s/EndermanOverhaul#71]
### Bug Description

The "Replace Default Enderman" option does not work, results in all vanilla enderman textures and animation being replaced with the one from this mod.

### How to Reproduce?

1. Go to config of the mod
2. Disable the "Replace Default Enderman" option
3. Find/spawn an enderman and see

### Expected Behavior

The texture and animation will no longer be replaced.

### Version

2.0.0

### Mod Loader Version

1.21.1 - 21.1.69

### Mod Loader

NeoForge

### Logs or additional context

[log](https://pastebin.com/qXq1Qdva)

### Code of Conduct

- [X] I have searched the issue tracker and confirmed that the issue has not been reported
- [X] I have checked the FAQ (if one is present) and my issue is not listed
- [X] I have verified that I am using the latest version of the mod
- [X] I have verified that I have installed all the required dependencies for the mod
- [X] I have verified that I do not have Optifine installed. We will close issues if we detect Optifine is in your pack. We cannot fix Optifine issues. Please do not report issues with Optifine present.

**Comment by Revv1e** [comment]
Same here, doing the same process as OP, and I cannot get the default enderman to show up...

**Comment by ScoutTheSaneless** [comment]
can also confirm

**Comment by Blabby1** [comment]
Yep, I personally prefer the default and this is kinda annoying. It also overrides fresh animations' animations, as well as any texture packs that change the endermans texture :/

**Comment by CaptConrad** [comment]
This ever gonna get fixed?
